system_tests/speech.py

```diff
@@ -109,3 +124,54 @@ def test_sync_recognize_gcs_file(self):
         result = self._make_sync_request(source_uri=source_uri,
                                          max_alternatives=1)
         self._check_best_results(result)
+
+    def test_async_recognize_local_file(self):
+        import time
+        with open(AUDIO_FILE, 'rb') as file_obj:
+            operation = self._make_async_request(content=file_obj.read(),
+                                                 max_alternatives=2)
+        retry_count = 100
+        while retry_count > 0 and not operation.complete:
+            retry_count -= 1
+            time.sleep(10)
+            operation.poll()
+
+        self.assertEqual(len(operation.results), 2)
+        self._check_best_results(operation.results)
+
+        results = operation.results
+        self.assertIsInstance(results[0], Transcript)
+        self.assertEqual(results[0].transcript, 'hello ' + self.ASSERT_TEXT)
+        self.assertGreater(results[0].confidence, 0.90)
```

Comment:
You already check all of lines 143, 144 and 145 in `self._check_best_results(operation.results)`